nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestWorkManagerObject.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour Al Kotob
+ */
+
+package org.nuxeo.rest.management;
+
+import static org.junit.Assert.assertEquals;
+import static org.nuxeo.rest.management.work.manager.WorkManagerObject.TIMEOUT_SECONDS_PARAM_KEY;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.automation.test.AutomationFeature;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.LogCaptureFeature;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.sun.jersey.core.util.MultivaluedMapImpl;
+
+/**
+ * @since 11.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features({ AutomationFeature.class, LogCaptureFeature.class })
+@Deploy("org.nuxeo.rest.management:OSGI-INF/test-work-dead-letter-queue.xml")
+public class TestWorkManagerObject extends ManagementBaseTest {
+
+    private static final String SUCCESS_KEY = "success";
+
+    private static final String TOTAL_KEY = "total";
+
+    public static final String PATH = "site/management/workmanager/runworksinfailure";
+
+    public static final String OK = "ok";
+
+    @Inject
+    LogCaptureFeature.Result logCaptureResult;
+
+    @Test
+    @LogCaptureFeature.FilterOn(logLevel = "WARN")
+    public void testRunningStatus() throws IOException {
+        // Test with timeout parameter
+        MultivaluedMap<String, String> formData = new MultivaluedMapImpl();
+        formData.add(TIMEOUT_SECONDS_PARAM_KEY, "10");
+        try (CloseableClientResponse response = httpClientRule.post(PATH, formData)) {
```

Comment:
I'm not sure this test is useful like that.
With the following:
```java
...
formData.add("foo", "bar");
try (CloseableClientResponse response = httpClientRule.post(PATH, formData)) {
...
```
The assertions of the test would not change.

I would say that we don't care about testing the timeout here, and stick only with the second test.

Reply:
I wanted to make sure I'm passing the parameter, but I only recall checking it...in the debugger.

But I think if it is possible, it should be checked that the api passes the params correctly.

Reply:
see my above suggestion on making two tests, i refactor the code to extract a new method ...